source/dbo/Views/Power BI Reporting/vwDashboardNotifications.sql

```diff
@@ -0,0 +1,23 @@
+﻿--this view supports the 'NTBS dashboard' report which shows users their cases which do not yet have a final treatment outcome 
+--applied to them. It's also used to display which notifications have which open alerts, and so may include cases
+--not yet in their service/region but where this is an open transfer request
+
+CREATE VIEW [dbo].[vwDashboardNotifications]
+	AS 
+	SELECT 
+		cd.NotificationId
+		,CASE 
+			WHEN overdueOutcome.NotificationId IS NOT NULL THEN 'Overdue'
+			WHEN DATEADD(DAY, -30, DATEADD(YEAR, 1, o.NotificationStartDate)) < GETUTCDATE() AND cd.TreatmentOutcome12months = 'No outcome recorded' THEN 'Nearly due'
+			WHEN DATEADD(DAY, -30, DATEADD(YEAR, 2, o.NotificationStartDate)) < GETUTCDATE() AND cd.TreatmentOutcome24months = 'No outcome recorded' THEN 'Nearly due'
+			WHEN DATEADD(DAY, -30, DATEADD(YEAR, 3, o.NotificationStartDate)) < GETUTCDATE() AND cd.TreatmentOutcome36months = 'No outcome recorded' THEN 'Nearly due'
+		END AS 'OutcomeStatus'
+		,openTransfer.AlertTBServiceCode
+		,tbsp.PHEC_Code AS AlertPhecCode
+	FROM [dbo].[Record_CaseData] cd
+	INNER JOIN [dbo].[RecordRegister] rr ON rr.NotificationId = cd.NotificationId AND rr.Denotified = 0
+	INNER JOIN [dbo].[Outcome] o ON o.NotificationId = cd.NotificationId
+	LEFT OUTER JOIN [dbo].[vwAlert] overdueOutcome ON overdueOutcome.NotificationId = cd.NotificationId AND overdueOutcome.AlertType LIKE '%DataQualityTreatmentOutcome%'
```

Comment:
There are a few notifications where the Notification Start Date would have changed if a relative date initially entered incorrectly so showing as Overdue when NotificationStartDate is now in 2022.

Reply:
Okay so there's an issue with the application not closing alerts when it should do, i.e. because the outcome is no longer overdue? I guess until that's resolved, this is at least consistent with the application, i.e. if you use the alerts visualisation on the left hand side of the dashboard, it will show the right records (rather than saying there are 16 alerts but only 15 records)?